lusSTR/scripts/marker.py

```diff
@@ -375,8 +386,17 @@ def convert(self):
         if len(self.uas_sequence) < 110:
             bracketed_form = collapse_repeats_by_length(self.uas_sequence, 4)
         else:
-            for m in re.finditer("GGGCTGCCTA", self.uas_sequence):
-                break_point = m.end()
+            if "GGGCTGCCTA" in self.uas_sequence:
+                break_point = self.uas_sequence.index("GGGCTGCCTA") + 10
+            else:
+                break_point = self.uas_sequence.index("TTTT") + 14
+            # for m in re.finditer("GGGCTGCCTA", self.uas_sequence):
+            #    break_point = m.end()
+            # try:
+            #    break_point
+            # except NameError:
+            #    for m in re.finditer("TTTT", self.uas_sequence):
+            #        break_point = m.end() + 10
```

Comment:
Sorry to be a stickler but this should be cleaned up before merging.

Reply:
GAH yes